docs/docs/usage.md

```diff
@@ -162,7 +162,6 @@ Generates the MTA deployment descriptor (`mtad.yaml` file) according to the prov
 
 <br>
 <br>
-
 <b>`mbt module-build`</b>
```

Comment:
The formatting of this line was lost, could you return the line break before it?

Reply:
fixed